tac/guidelines/project-consistency-guidelines.md

```diff
@@ -0,0 +1,111 @@
+
+This document outlines consistent approaches to certain aspects of running and maintaining an LFDT project.
+
+LFDT projects will always have their own approaches to how they manage and govern themselves, how they
+communicate and collaborate, and how they track and communicate progress.
+
+The LFDT already sets out `MUST` and `SHOULD` and `MAY` behaviours that projects should treat accordingly. This document
+is not intended to replace any of the existing governance rules set out by the LFDT. Instead it is designed improve
+consistency across projects such that inter-project collaboration is easier, and to make it easier for individuals to
+collaborate to multiple LFDT projects.
+
+**MRW** - There is potentially a lot of overlap with [project best practices](./project-best-practices.md)
+
+**MRW** - Perhaps this document should be wrapped into that document?
+
+## Relationship to existing guidelines
+
+ - MRW: Suggest some discussion around consistency vs guidelines
+
+### Areas of consistency by stakeholder
+
+1. LFDT
+   - Communication
+      - Release activity
+   - CVE bounties
+3. Project users
+   - Communication
+      - **Readme structure that gives me a quick overview of what the project is/does** 
+      - Release activity
+      - **CVE resolution should be included in release notes including specific CVE numbers**
+         - (Potentially GH built-in CVE labelling)
+      - CVE advisories
+   - Issue tracking
+      - Progress on issues & bug fixes
+   - Roadmap
+      - Where to find it
+      - How to contribute to it (raise issues? attend contributor calls?)
+5. Project maintainers & contributors
+   - Communication
+      - **Readme structure that gives me a quick way to learn how to set up a development environment**
+      - Release planning
+         - **Public project tracking board**
+            - **Github projects for tracking**
+            - Alternatives acceptable, as long as public and not requiring authentication to view
+         - **Project tracking board linked from repo landing page**
+   - CVE reporting
+      - **Raising GH advisories for at least HIGH and CRITICAL CVEs??**
+   - Bug tracking
+   - Roadmap
+      - Where to find it 
+
+
+---
+**OLD NOTES BELOW**
+---
+
+### Communication
+
+### Bug tracking
+
+ - MRW: There is already guidance about using Github practices such as "bug: ..." issue naming
+
+### Release planning
+
+ - MRW: Post on the "maintainers" or "contributors" discord channel that a new release is planned with an approximate timeline
+   - MRW: Is this being too prescriptive?
+
+### Release communication
+
+ - MRW: Use Github releases to communicate new releases
+ - MRW: Post on the main project discord channel (i.e. not just the "maintainers" or "contributors" channels
+
+### Technical discussion that contributes to project direction
+
+ - Where is discussion, context and background that helps understand story prioritization and the current roadmap
```

Comment:
@matthew1001 I recommend mentioning here another pet peeve of mine: commit messages should contain the full context of the change such as the why, how, what, etc. The reason being is that if the repo ever have to be migrated to a different platform most likely the GitHub PR reviews will not travel along with the repo. This kind of migration seems fairly low probability nowadays, but I recommend it to people nonetheless.